environments/prod/courttribunalfinder-service-gov-uk.yml

```diff
@@ -0,0 +1,21 @@
+ns: []
+
+mx: []
+
+A:   
+  - name: "@"
+    ttl: 3600
+    record:
+      - "51.105.37.227"
+
+txt: 
+  - name: "_smtp._tls"
+    ttl: 3600
+    record: 
+      - "v=TLSRPTv1;rua=mailto:tls-rua@mailcheck.service.ncsc.gov.uk"
+
+cname:
+  - name: "hw2mzkztnolp"
+    ttl: 300
+    record: 
```

Comment:
i don't think this is it, cnames can only have one target:
https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/apply-for-probate-service-gov-uk.yml#L17